frontends/ol-components/src/components/LearningResourceExpanded/LearningResourceExpanded.tsx

```diff
@@ -99,6 +99,8 @@ const CallToAction = styled.div`
 const StyledLink = styled(ButtonLink)`
   text-align: center;
   width: 220px;
+  font-size: ${({ theme }) => theme.typography.pxToRem(14)};
+  padding: 12px;
```

Comment:
What's this padding for? 

ButtonLink is one of our "design system" components....I tend to think of CSS on an element as (A) "styling the element" [example: padding, color, font size] or (B) "positioning + sizing the element in the flow of the document" [example: margin, width]. I generally would not expect design systems components to need much Type A styling. 


**Request:** 

Could you try removing the font size / padding and change `size="medium"` down below? The medium sized button should the correct font size, etc.
```tsx
      <StyledLink
        target="_blank"
        size="large"
        href={getCallToActionUrl(resource) || ""}
      >
```

Reply:
ah perfect. That did the trick. I was looking for the proper way to get this to work with the longer "access course materials" text but didnt stumble on it.